assets/locales/debugger.properties

```diff
@@ -273,9 +273,9 @@ sourceTabs.closeTab=Close tab
 # for closing the other tabs.
 sourceTabs.closeOtherTabs=Close others
 
-# LOCALIZATION NOTE (sourceTabs.closeTabsToRight): Editor source tab context menu item
-# for closing the tabs to the right of the selected tab.
-sourceTabs.closeTabsToRight=Close tabs to the right
+# LOCALIZATION NOTE (sourceTabs.closeTabsToEnd): Editor source tab context menu item
+# for closing the tabs to the end of the selected tab.
+sourceTabs.closeTabsToEnd=Close tabs to the end
```

Comment:
Yes, so the string ID should be "to the end" but they text should "to the right", at least for english where the language is LTR (left to right).  This allows the translator to choose what "the end" means based on the language.